content/en-us/docs/maintain/collator copy.md

```diff
@@ -0,0 +1,136 @@
+---
```

Comment:
better rename this file and remove the `copy` suffix

Reply:
@shelvenzhou yes, this file should be renamed to 'gatekeeper'.

Reply:
renamed